CockroachDB Cloud Billing with AWS Marketplace (#18690)
cockroachdb/docs: src/current/cockroachcloud/billing-management.md

The **Billing** page contains an overview of your charges and the payment details on file for your CockroachDB {{ site.data.products.cloud }} organization. To view the **Billing** page, [log in](https://cockroachlabs.cloud/) and click **Billing**.

## Set up billing for an organization

Users with the [Billing Coordinator]({% link cockroachcloud/authorization.md %}#billing-coordinator) role can manage billing for the organization. 

> You can manage your CockroachDB {{ site.data.products.cloud }} payments by using a credit card or through an [AWS Marketplace](https://aws.amazon.com/marketplace) subscription for easy integration with your existing AWS billing.

Reviewer: ```suggestion
You can pay for CockroachDB {{ site.data.products.cloud }} by using a credit card or [AWS Marketplace](https://aws.amazon.com/marketplace), along with other AWS charges.
```

Your credit card will be charged after the trial ends.
{{site.data.alerts.end}}

</section>

<section class="filter-content" markdown="1" data-scope="marketplace">

## Subscribe through AWS Marketplace

To subscribe to CockroachDB {{ site.data.products.cloud }} through the AWS Marketplace:

> 1. Navigate to the [CockroachDB product page](https://aws.amazon.com/marketplace/pp/prodview-3zbkzekdohwly) on the AWS Marketplace and click **View purchase options**.

Reviewer 2: The link to the new AWS listing is https://aws.amazon.com/marketplace/pp/prodview-n3xpypxea63du

1. On the **Subscribe to CockroachDB Cloud** page, click **Subscribe** and then click **Set up your account**. You will be directed to the CockroachDB {{ site.data.products.cloud }} console.
1. [Register]({% link cockroachcloud/create-an-account.md %}#register-a-new-account) a new CockroachDB {{ site.data.products.cloud }} account or sign in to your existing account.

> 1. Select the organization CockroachDB {{ site.data.products.cloud }} you want to subscribe through the AWS Marketplace and click **Subscribe to AWS Marketplace**.

Reviewer 2: Maybe consider adding a disclaimer here because not all your orgs will be eligible for connecting to the AWS Marketplace. 

Orgs already linked to a marketplace subscription and invoice orgs are ineligible. You must also have the Billing Coordinator role in the org to link it.

Reviewer 2: It might be good to highlight eligibility requirements, as this is one area a user may get confused. We have a error message in the UI that says you can read more in the docs about eligibility

## Unsubscribe from CockroachDB {{ site.data.products.cloud }}

> Before you unsubscribe from CockroachDB {{ site.data.products.cloud }}, [add a credit card](#add-a-credit-card) as a backup payment method to avoid cluster deletion.

Reviewer 2: You can't add a credit card _before_ unsubscribing. You should be prepared to add one after unsubscribing. If there's no alternate payment method in 6 hours, we will delete your clusters, terminating service.

To unsubscribe from CockroachDB {{ site.data.products.cloud }}:

> 1. Navigate to the [AWS Marketplace](https://aws.amazon.com/marketplace) and then to the **Manage Subscriptions** page. From the list of subscriptions, select **CockroachDB Cloud**.

Reviewer 2: The name of the listing is "CockroachDB (pay-as-you-go)".

> 1. Click **Actions** in the Agreement section. From the dropdown, click **Cancel Subscription**.

Reviewer 2: Just FYI, you need to "confirm" cancellation. The user should see this on their screen, so maybe don't need to mention this.

Your credit card will be charged after the trial ends.
{{site.data.alerts.end}}

</section>

<section class="filter-content" markdown="1" data-scope="marketplace">

## Subscribe through AWS Marketplace

To subscribe to CockroachDB {{ site.data.products.cloud }} through the AWS Marketplace:

> 1. Open to the [CockroachDB (pay-as-you-go)](https://aws.amazon.com/marketplace/pp/prodview-n3xpypxea63du) page on the AWS Marketplace and click **View purchase options**.

Reviewer: ```suggestion
1. From the [AWS Marketplace page for CockroachDB (pay-as-you-go)](https://aws.amazon.com/marketplace/pp/prodview-n3xpypxea63du),  click **View purchase options**. to open the **Subscribe to CockroachDB Cloud** page
```

> 1. On the **Subscribe to CockroachDB Cloud** page, click **Subscribe** and then click **Set up your account**. You will be directed to the CockroachDB {{ site.data.products.cloud }} console.

Reviewer: ```suggestion
1. Click **Subscribe**, then click **Set up your account**. You will be directed to the CockroachDB {{ site.data.products.cloud }} console.
```

1. [Register]({% link cockroachcloud/create-an-account.md %}#register-a-new-account) a new CockroachDB {{ site.data.products.cloud }} account or sign in to your existing account.

> 1. Select the CockroachDB {{ site.data.products.cloud }} organization that you want to subscribe through the AWS Marketplace and click **Subscribe to AWS Marketplace**.

Reviewer: ```suggestion
1. If you have access to multiple CockroachDB {{ site.data.products.cloud }} organizations, select an organization to update its billing configuration.

    {{site.data.alerts.callout_info}}
    Organizations that are configured for invoice billing or are already linked to AWS Marketplace are not listed.
    {{site.data.alerts.end}}

    
1. Click **Subscribe to AWS Marketplace**.
```

> {{site.data.alerts.callout_info}}
> Only eligible CockroachDB {{ site.data.products.cloud }} organizations are listed. To be eligible, the org cannot already be:
> - Linked to a marketplace subscription.
> - Configured for invoice billing.
> {{site.data.alerts.end}}

Reviewer: ```suggestion
```

> ## Unsubscribe from CockroachDB {{ site.data.products.cloud }}

Reviewer: ```suggestion
## Unsubscribe from AWS Marketplace
```

> Before you unsubscribe from CockroachDB {{ site.data.products.cloud }}, if you wish to retain the subscribed org's clusters, be prepared to [add a credit card](#add-a-credit-card) immediately after you unsubscribe, to avoid cluster deletion.

Reviewer: ```suggestion
{{ site.data.alerts.callout_danger }}
After you unsubscribe your CockroachDB {{ site.data.products.cloud }} from AWS Marketplace billing, clusters will be deleted and applications may be disrupted unless you [add a credit card](#add-a-credit-card) within 6 hours after you unsubscribe from AWS Marketplace billing.
{{ site.data.alerts.end }}
```

> {{ site.data.alerts.callout_danger }}
> If no alternate payment method is specified within 6 hours of unsubscribing through the marketplace, your clusters will be deleted, terminating service.
> {{ site.data.alerts.end }}

Reviewer: ```suggestion
```

To unsubscribe from CockroachDB {{ site.data.products.cloud }}:

> 1. Go to the [AWS Marketplace](https://aws.amazon.com/marketplace) and then open the **Manage Subscriptions** page. From the list of subscriptions, select **CockroachDB (pay-as-you-go)**.

Reviewer: ```suggestion
1. From [AWS Marketplace](https://aws.amazon.com/marketplace), click **Manage Subscriptions**.
1. In the list of subscriptions, click **CockroachDB (pay-as-you-go)**.
```

> 1. In the **Agreement** section, click **Actions**, then click **Cancel Subscription**. Review and accept the confirmation dialog to complete the cancellation.

Reviewer: ```suggestion
1. In the **Agreement** section, click **Actions**, then click **Cancel Subscription**. Review.
1. Click **Accept** to unsubscribe.
```
